### PR TITLE
Fix: Security for recovery phrase

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Related task: #257 
### Checklist (If you won't pass this checklist please comment why)

-   [ ] I removed all commented or unnecessary code.
-   [ ] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: client/src/components/web-extension/RecoveryPhrase/index.jsx [hunk 1]
```diff
@@ -26,38 +28,64 @@ const RecoveryPhrase = () => {
...
+								<li className="cd_we_recovery-keyphrase__word" key={`left-${index}`}>
+									<span className="counter">{index + 1}</span>
+									<span className="value">{word}</span>
+								</li>
+							))}
+						</ul>
+						{
+							isBlurred && (
+								<div className="cd_we_recovery-keyphrase__blur-overlay">
+									Click to reveal secret recovery phrase
```
Comment (duc-q) [inline client/src/components/web-extension/RecoveryPhrase/index.jsx:54]: Should be "Reveal secret recovery phrase"

File: client/src/components/web-extension/RecoveryPhrase/RecoveryPhrase.scss [hunk 2]
```diff
@@ -6,6 +6,30 @@
...
+		&-overlay {
+			position: fixed;
+			// Center postion
+			top: 41%;
+			left: 50%;
+			transform: translate(-50%, -50%);
+			font-size: 14px;
+			width: 260px;
+			color: #777e91;
+			// font-weight: 700;
```
Comment (duc-q) [inline client/src/components/web-extension/RecoveryPhrase/RecoveryPhrase.scss:22]: Can be removed

File: client/src/components/web-extension/CreateWallet/RecoveryPhrasePage.jsx [hunk 3]
```diff
@@ -7,9 +7,9 @@ import { generateKeyphrase, setNextStep } from '@cd/actions/createWalletActions'
 import SelectEncryptionType from '@cd/web-extension/Common/SelectEncryptionType';
 import NumberRecoveryWordsSelect from '@cd/web-extension/Common/NumberRecoveryWordsSelect';
 import { selectCreateWalletKeyphraseAsMap, selectCreateWalletKeyphrase } from '@cd/selectors/createWallet';
-import { DownloadButton } from '@cd/components/web-extension/Common/DownloadButton';
 import { NUMBER_OF_RECOVERY_WORDS } from '@cd/constants/key';
 import './RecoveryPhrasePage.scss';
+import CopyButton from '../Common/CopyButton/index';
```
Comment (duc-q) [inline client/src/components/web-extension/CreateWallet/RecoveryPhrasePage.jsx:12]: Should be above `import './RecoveryPhrasePage.scss';`

**Review by duc-q — APPROVED** [review]
Just minor feedbacks

**Comment by djhahe** [comment]
image
@hoailinh1210 User can copy phrase without reveal , please help to move the copy button outside sensor section

**Comment by djhahe** [comment]
@hoailinh1210  Please apply reveal for viewing secret key as well

File: client/src/components/web-extension/RecoveryPhrase/index.jsx (hunk 1 above)
Comment (hoailinh1210) [reply client/src/components/web-extension/RecoveryPhrase/index.jsx:54]: Maybe `Click` is more clear for user :D

**Comment by hoailinh1210** [comment]
Hi @djhahe , I updated all our feedback,
Please help me review on it, thanks  👍

**Review by djhahe — APPROVED** [review]

**Pull request merged**